server/routes/api/v1/petsRouter.js

```diff
@@ -13,7 +13,21 @@ petsRouter.get("/", async (req, res) => {
   }
 })
 
-petsRouter.get("/type", async (req, res) => {
+petsRouter.get("/:id", async (req, res) => {
+  try {
+    const pet = await Pet.findById(req.params.id)
+    res.status(200).json({ pet: pet })
+  }
+  catch (error) {
+    let statusCode = 500
+    if (error.constructor.name === "TypeError") {
+      statusCode = 404
+    }
+    res.status(statusCode).json({ errors: error })
+  }
+})
+
+petsRouter.get("/types", async (req, res) => {

```

Comment:
Looks like this endpoint should live in the router specifically for `types`, rather than the path reading `pets/types`. If we want all the _types_ of pets we can go through the `api/v1/types` router. 

I'm unsure what the difference is between `Pet.findAllTypes()` and `Type.findAll()` as I don't see a model for Type. It looks like the query for `Pet.findAllTypes()` should instead be querying `Type.findAll()`, which would confirm moving the endpoint to that router